### ISSUE TITLE
[Bug]: iOS Compilation error with version 0.14.2 after enforcing minimal iOS platform ver to 11.0

### ISSUE THREAD

**Issue by yanivshaked** [oddbit/flutter_facebook_app_events#156]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

I have changed the minimal iOS platform to 11.0 in my `Podfile`:
```
platform :ios, '11.0'
deployment_target = '11.0'
```
Also updated the minimal deployment version in Xcode.
When building the project, I'm getting the following error messages:
```

❌  /Users/twelvetwelve/.pub-cache/hosted/pub.dartlang.org/facebook_app_events-0.14.2/ios/Classes/SwiftFacebookAppEventsPlugin.swift:91:98: cannot convert value of type '[String : Any]' to expected argument type '[AppEvents.ParameterName : Any]'

            AppEvents.logEvent(AppEvents.Name(eventName), valueToSum: valueToDouble, parameters: parameters)
                        ^



❌  /Users/twelvetwelve/.pub-cache/hosted/pub.dartlang.org/facebook_app_events-0.14.2/ios/Classes/SwiftFacebookAppEventsPlugin.swift:93:71: cannot convert value of type '[String : Any]' to expected argument type '[AppEvents.ParameterName : Any]'

            AppEvents.logEvent(AppEvents.Name(eventName), parameters: parameters)
                                                                                                 ^


```

### Expected Behavior

Build should be successful.

### Steps To Reproduce

Using Xcode 13.1, flutter version 2.5.3:
1. Change minimal version of your project to iOS version 11.
2. Build the project.

### Platform

iOS

### Flutter Doctor

```shell
[✓] Flutter (Channel stable, 2.5.3, on macOS 11.4 20F71 darwin-arm, locale en-IL)
    • Flutter version 2.5.3 at /Users/twelvetwelve/Development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (3 weeks ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.


[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 13.1, Build version 13A1030d
    • CocoaPods version 1.11.2

[✗] Chrome - develop for the web (Cannot find Chrome executable at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).

[✓] VS Code (version 1.54.3)
    • VS Code at /Users/twelvetwelve/Downloads/Visual Studio Code.app/Contents
    • Flutter extension version 3.22.0

[!] Connected device
    ! No devices available

! Doctor found issues in 4 categories.
```

**Comment by RicardoRB** [comment]
Same problem here, but I'm using:

`platform :ios, '10.0'`

I don't think the target version is related but the type param:

`SwiftFacebookAppEventsPlugin.swift:91:98: error: cannot convert value of type '[String : Any]' to expected argument type '[AppEvents.ParameterName : Any]'`

**Comment by yanivshaked** [comment]
I agree, but I'm not sure why this error came up along with the minimal iOS version change. (Build was working fine up to that point)

**Comment by yanivshaked** [comment]
@RicardoRB You are welcome to try:
```
  facebook_app_events:
    git:
      url: https://github.com/yanivshaked/flutter_facebook_app_events.git
      ref: "fbc72fb"
```
If it works, I will issue a PR.

**Comment by RicardoRB** [comment]
Did it work for you? It seems like the right answer, but I don't know if now I can send a string, and it would be converted as AppEvents.ParameterName

**Comment by yanivshaked** [comment]
It should work, I'm testing it

**Comment by RicardoRB** [comment]
It worked to me 👌

**Comment by yanivshaked** [comment]
👍

**Comment by DennisAlund** [comment]
Thank you for your contribution. PR #157 has been merged and is released in [`v0.14.3`](https://pub.dev/packages/facebook_app_events/changelog#0143)
